src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/CreateLegacyContactOnVisitRequestDto.kt

```diff
@@ -1,12 +1,25 @@
 package uk.gov.justice.digital.hmpps.visitscheduler.dto
 
+import com.fasterxml.jackson.annotation.JsonIgnore
 import io.swagger.v3.oas.annotations.media.Schema
-import javax.validation.constraints.NotBlank
-
-data class CreateLegacyContactOnVisitRequestDto(
-  @Schema(description = "Contact Name", example = "John Smith", defaultValue = "UNKNOWN", required = false)
-  @field:NotBlank
-  val name: String? = "UNKNOWN",
-  @Schema(description = "Contact Phone Number", example = "01234 567890", defaultValue = "UNKNOWN", required = false)
-  val telephone: String? = "UNKNOWN",
-)
+import uk.gov.justice.digital.hmpps.visitscheduler.validation.NullableNotBlank
+
+data class CreateLegacyContactOnVisitRequestDto(@JsonIgnore val _name: String? = UNKNOWN, @JsonIgnore val _telephone: String? = UNKNOWN) {
```

Comment:
class CreateLegacyContactOnVisitRequestDto @JsonCreator constructor(
Would something like that work